pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -516,906 +513,6 @@ func TestPodFitsResources(t *testing.T) {
 
 }
 
-func TestPodFitsHost(t *testing.T) {
```

Comment:
Hmm... Can we move it somewhere instead of deleting them?

Reply:
So those already exist in the corresponding plugins, but the reason I am deleting them is because the only public function here is the GeneralPredicate, which provide the necessary coverage.